docs/welcome.md

```diff
@@ -0,0 +1,69 @@
+
+## Food
+
+The "kitchen" operates on a take-one-leave-one basis, where the space doesn't contribute directly to any food or supplies, but members volunteer to restock items that they consume.  Although there is some limited cooking equipment, the kitchen isn't set up for making meals.  If you do get any of the equipment dirty, please wipe it down and wash the things you used in the industrial wash basin in the back of the space.
+
+## Parking Tickets
+
+If you leave any gear at the space, get a yellow parking ticket request form from the office, fill it out, tag the equipment, and message the board on Telegram.  They'll attach the appropriate paperwork to the item, including an expiration date.  Storing things in the upstairs area generally requires coordination with other people; message the telegram group if you do this, and make sure to fill out a parking ticket request.
+
+## HVAC
+
+There's a Nest thermostat in the kitchen room, near the inner door.  Just twist the outer ring of the thermostat to set the desired temperature, and The Cloud will try to accomodate your request.
+
+## The Wifis
+
+Scan the big QR code in the office to get the current access information.  The WPA pre-shared key should also be printed on that same QR code.
+
+## Safety Third!
+
+It's assumed that you know what you're doing, so there aren't many safeguards on any of the equipment.  Some of it could maim or blind you.  If you don't already know you can operate something safely, you can't and you shouldn't.  Message the telegram channel for assistance.  Maybe you'll make a new friend!
+
+## Breakers
+
+Some of the gear can and will trip circuit breakers when operated in certain combinations.  Example: the toaster and the microwave in the kitchen, or the upright router and a table saw on one of the overhead extension cords, and probably others.  You can reset the overhead breakers using one of the long sticks in the corner, to the left of the rolling door.  Other breakers just need a boop in the right place, look around for a reset button.
+
+## TV
+
+The TV near the conference table has a Chromecast normally connected to its HDMI port, although sometimes it's swapped out for the Chromebox used to run meetings.  There's also a VGA cable on the conference table that you can use to hook up your own laptop.
+
+## Speakers
+
+The sound system can be driven by any bluetooth client, just go to the office and hit the PAIRING button on the remote for the Sony amplifier, then pair your device like usual.  There's also a Chrome streaming device hooked up to the AUX port, and you can stream to that over the network if you set the amp to the AUX input.  Try not to blast audio too loudly.
+
+## Cat
+
+There's some cats that live in the parking lot.  The neighbors feed them.  Don't bother trying to make friends with them, they don't like humans getting too close.
+
+## HAZMAT
+
+(there should be something from the board here)
```

Comment:
I don't know what we'd say. I know amp has set us up in some way with the state, I'll ask them.

Reply:
We are registered as a [very small quantity generator](https://hhw.sccgov.org/very-small-quantity-generator-disposal-program) with the county.  There are labels near the only hazmat we have so far (resin-inundated IPA) in the corner by the shower.  There is an associated fee so disposal needs to be coordinated by Amp.